apps/telegram/app/(general)/scout/page.tsx

```diff
@@ -6,12 +8,14 @@ export default async function Scout({
   searchParams: { [key: string]: string | string[] | undefined };
 }) {
   const scoutSort = (searchParams.scoutSort as string) || 'points';
-  const builderSort = (searchParams.builderSort as string) || 'rank';
-  const builderOrder = (searchParams.builderOrder as string) || 'asc';
+  const builderSort = (searchParams.builderSort as BuildersSortBy) || 'week_gems';
```

Comment:
Why isn't week_gems part of BuildersSortBy?

Reply:
BuildersSortBy does contain `week_gems`